src/hostcalls.rs

```diff
@@ -319,7 +319,14 @@ fn get_hostfunc(
                         .get_expect_get_property(path_raw)
                     {
                         Some(expect_property_value) => expect_property_value,
-                        None => vec![],
+                        None => {
+                            println!(
+                                "[vm->host] proxy_get_property(path_data, path_size) -> NotFound, status: {:?}",
+                                get_status()
+                            );
+                            assert_ne!(get_status(), ExpectStatus::Failed);
```

Comment:
`get_expect_get_property(path_raw)` is setting status, so it must be asserted before returning.